ccutils.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+
+cc::print_error() {
```

Comment:
It is just some quasi-namespacing to try to avoid conflicting with functions the sourcee (?) may have already defined.